package.json

```diff
@@ -58,6 +58,7 @@
     ]
   },
   "dependencies": {
+    "dompurify": "^1.0.8",
```

Comment:
Can you remove the `^` here?

Reply:
Wouldn't it be ironic if RES suffered a vulnerability thanks to an invisible upgrade to a compromised version of a security package.

I added the integrity sha to yarn.lock, too.